nominations/alexey-alekhin.md

```diff
@@ -0,0 +1,24 @@
+# Alexey Alekhin
+
+## Proposer
+
+This candidate has been nominated by Ólafur Páll Geirsson and seconded by *seconder*.
+
+## Qualifications and Experience
+
+- Author of [atom-ide-scala](https://github.com/laughedelic/atom-ide-scala)
+- Maintainer of [Metals](https://github.com/scalameta/metals)
```

Comment:
Could you also mention [atom-sbt-client](https://github.com/laughedelic/atom-sbt-client)? It's an (LSP) integration of the sbt server with Atom.

I'm not sure if it counts as relevant tooling projects, but I also authored some sbt plugins:
* [sbt-publish-more](https://github.com/laughedelic/sbt-publish-more)
* [sbt-s3-resolver](https://github.com/ohnosequences/sbt-s3-resolver)
* [sbt-github-release](https://github.com/ohnosequences/sbt-github-release)
* [literator](https://github.com/laughedelic/literator)

Reply:
all 5 of those looks relevant and noteworthy, to me!

Reply:
I agree! The template specifically mentions that max 5 bullets are allowed, which ones would you like to list @laughedelic ?

Reply:
I guess, the most used 😄 
* sbt-s3-resolver
* sbt-github-release

<!-- --> + atom-sbt-client, because I think it's more relevant to the LSP topic

Reply:
Feel free to bend the rules and group some of the SBT plugins together into a single bullet point.